workers/unity/Packages/com.improbable.gdk.core/Worker/WorkerConnector.cs

```diff
@@ -415,6 +413,11 @@ private void OnDisconnected(string reason)
 
         private IEnumerator DeferredDisposeWorker()
         {
+            if (Worker?.World != null)
+            {
+                PlayerLoopUtils.RemoveFromPlayerLoop(Worker.World);
```

Comment:
how about moving that to the `Dispose` inside the `Worker`.

Reply:
It needs to be done the frame before the worker is disposed, that's why its there.

Reply:
^might be worth adding that as a comment in code